http/http.yaml.example

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: cloud-build-notifiers/v1alpha1
+apiVersion: github.com/GoogleCloudPlatform/cloud-build-notifiers/v1
```

Comment:
I see that this is similar to a Kubernetes spec (though I'm not sure if this is a full-fledged CRD).

Worth noting is that this apiVersion type is not Kubernetes compatible with it's use of subpaths. See
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/api-group.md and
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md#identifiers-and-names-in-kubernetes for more details.

Some alternatives:
- cloud-build-notifiers/v1 (keep it as is)
- notifiers.cloudbuild.googleapis.com/v1 (stays consistent with API)

Reply:
Given that the config isn't part of the GCB API at the moment, I'll go with the first one.